docs/RELEASE_NOTES.md

```diff
@@ -42,8 +42,12 @@
 [Javadoc](https://commercetools.github.io/commercetools-sync-java/v/1.5.0/) | 
 [Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/1.5.0)
 
-- 🎉 **New Features** (1)
```

Comment:
Actually, this is left on purpose for the next release 😉 We always leave one example for the next person doing the release.

Reply:
yes. 2 is there, because:
1. Will be un-commented so that is used the new release's notes.
2. The other will be left commented (as an example) for the next release (after the new release).

Reply:
Why we need an example? Isn't it obvious from existing release notes, also we will have a commented version, I am asking why we need 2 commented release notes part? anyway we can skip this which is not related to that PR.

Reply:
We usually always have one as an example, because it's just easier. But in this case, it's 2, just because I know I am realising very soon.